st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
Well, otherwise, in a distributed system we would need a coordinator if we wanted to ensure this piece of bootrstrap code was executed and this makes implementation and everything else more complex (and potentially error prone).

Relying on the idempotent nature of operation (where possible) and potentially calling it multiple times, is imo a better and cleaner approach. It might have a small performance penalty / increase startup time, but services are long running so shouldn't be a big deal in this case :)